docs/sources/flow/reference/components/prometheus.exporter.unix.md

```diff
@@ -144,7 +146,7 @@ The following blocks are supported inside the definition of
 `fs_types_exclude` defaults to the following regular expression string:
 
 ```
-^(autofs\|binfmt_misc\|bpf\|cgroup2?\|configfs\|debugfs\|devpts\|devtmpfs\|fusectl\|hugetlbfs\|iso9660\|mqueue\|nsfs\|overlay\|proc\|procfs\|pstore\|rpc_pipefs\|securityfs\|selinuxfs\|squashfs\|sysfs\|tracefs)$
+^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$
```

Comment:
@rfratto Can you take a look at this?

Reply:
LGTM, I'm not sure why they were escaped in the first place and it doesn't look like they should've been since we weren't trying to literally match the `|` character.